api/apps/api/src/modules/access-control/scenarios-acl/scenario-acl.controller.ts

```diff
@@ -0,0 +1,58 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  ForbiddenException,
+  Query,
+  Patch,
+  HttpCode,
+  HttpStatus,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ScenarioAclService } from '@marxan-api/modules/access-control/scenarios-acl/scenario-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import {
+  ApiBearerAuth,
+  ApiNoContentResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import { isLeft } from 'fp-ts/lib/These';
+import {
+  UserRoleInScenarioDto,
+} from '@marxan-api/modules/access-control/scenarios-acl/dto/user-role-scenario.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Scenarios-Users Roles')
+@Controller(`${apiGlobalPrefixes.v1}/roles/scenarios`)
```

Comment:
But that is the root url for the endpoints under this module on Swagger, the url in issue description is in line 36. `  @Patch(':scenarioId/users')
`

Also tests will validate the logic (added new users with roles and changed roles) when PR 714 is merged as the only way to check if users are or not in a project was with the endpoint added in there (that's why the comment I added for this PR on the tests).

Maybe it was not a good idea to cut into pieces the work to be done in the controller/service because it ends up with cases like this where something needs a previous piece of work to run "effectively"